docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -3,29 +3,33 @@ This guide is maintained in the main Quarkus repository
 
-The `quarkus-keycloak-authorization` extension is based on `quarkus-oidc` and provides a policy enforcer that enforces access to protected resources based on permissions managed by Keycloak and currently can only be used with the Quarkus xref:security-oidc-bearer-token-authentication.adoc[OIDC service applications].
+The `quarkus-keycloak-authorization` extension relies on `quarkus-oidc`. It includes a policy enforcer that regulates access to secured resources. Access is governed by permissions set in Keycloak. Currently, this extension is compatible solely with Quarkus xref:security-oidc-bearer-token-authentication.adoc[OIDC service applications].
 
 It provides a flexible and dynamic authorization capability based on Resource-Based Access Control.
 
-Instead of explicitly enforcing access based on some specific access control mechanism such as Role-Based Access Control(RBAC), `quarkus-keycloak-authorization` checks whether a request is allowed to access a resource based on its name, identifier or URI by sending a bearer access token verified by `quarkus-oidc` to Keycloak Authorization Services where an authorization decision is made.
+Rather than explicitly enforcing access through specific mechanisms such as role-based access control (RBAC), `quarkus-keycloak-authorization` determines request permissions based on resource attributes such as name, identifier, or Uniform Resource Identifier (URI). This process involves sending a `quarkus-oidc`-verified bearer access token to Keycloak Authorization Services for an authorization decision.
 
-Use `quarkus-keycloak-authorization` only if you work with Keycloak and have Keycloak Authorization Services enabled to make authorization decisions. Use `quarkus-oidc` if you do not work with Keycloak or work with Keycloak but do not have its Keycloak Authorization Services enabled to make authorization decisions.
+Use `quarkus-keycloak-authorization` only if you work with Keycloak and have Keycloak Authorization Services enabled to make authorization decisions.
+Use `quarkus-oidc` if you do not work with Keycloak or work with Keycloak but do not have its Keycloak Authorization Services enabled to make authorization decisions.
 
-By externalizing authorization from your application, you are allowed to protect your applications using different access control mechanisms as well as avoid re-deploying your application every time your security requirements change, where Keycloak will be acting as a centralized authorization service from where your protected resources and their associated permissions are managed.
+By shifting authorization responsibilities outside your application, you enhance security through various access control methods while eliminating the need for frequent re-deployments whenever security needs evolve.
+Keycloak serves as a centralized authorization hub, managing your protected resources and their corresponding permissions effectively.
```

Comment:
Sometimes moving some content into independent sentences nearly changes the message - this is just a general observation. 
IMHO something like `In this case, Keycloak acts as...` would be a bit better, otherwise it is just a standalone message without an obvious link to the prev statement

Reply:
Updated. Thank you.